index.js

```diff
@@ -21,7 +21,7 @@ function versionCheck(hbCompiler, hbRuntime) {
 function getLoaderConfig(loaderContext) {
   var query = loaderUtils.getOptions(loaderContext) || {};
   var configKey = query.config || 'handlebarsLoader';
-  var config = loaderContext.options[configKey] || {};
+  var config = loaderContext.rootContext[configKey] || {};
```

Comment:
Or would it rather make sense to replace this with something along the lines of `loaderContext.rootContext ?
 loaderContext.rootContext[configKey] : loaderContext.options[configKey]` to make it backwards-compatible?

Reply:
yeah, backwards compatible would be better.

Reply:
any updates on this?

Reply:
Sorry for dropping the ball here, will update it today.

Reply:
@pcardune Done